docs/channels.md

```diff
@@ -0,0 +1,760 @@
+```
+
+Get the channels our ship is in and their contents. Returns `channels`
+
+[channels](#channels-1)
+#### /unreads
+```hoon
+=c -build-file /=groups=/sur/channel/hoon
+.^(unreads:c %gx /=channels=/unreads/channel-unreads)
+```
+
+Get unread information for the channels our ship is in. Returns `unreads`
+
+[unreads](#unreads-2)
+#### /init
+```hoon
+=c -build-file /=groups=/sur/channel/hoon
+.^([unreads:c channels:c] %gx /=channels=/init/noun)
+```
+
+Combination of `/channels` and `/unreads`. Returns `[unreads channels]`
+
+[unreads](#unreads-2) \| [channels](#channels-1)
+### Pokes
+#### Create a Group
+```hoon
+=g -build-file /=groups=/sur/groups/hoon
+
+=create :*
+  name=%mygroup
+  title='My group'
+  description='This is my group'
+  image=''
+  cover=''
+  cordon=`cordon:g`[%open ban=[ships=~ ranks=~]]
+  members=`(jug ship sect:g)`(~(put ju *(jug ship sect:g)) our %admin)
+  secret=%.n
+  ==
+
+:groups &group-create create
+```
+
+This is done with `%groups` instead of `%channels`, but it's necessary for testing. 
+
+#### %channel-action
+
+Takes an [a-channels](#a-channels), which can be one of several actions. Each action is covered below.
+
+##### %create
+```hoon
+=c -build-file /=groups=/sur/channel/hoon
+=g -build-file /=groups=/sur/groups/hoon
+
+=create-channel :*
+  %chat
+  %mychat
+  [our %mygroup]
+  title='My chat channel'
+  description='Chat about things'
+  readers=~
+  writers=~
+==
+
+:channels &channel-action [%create create-channel]
+```
+
+[create-channel](#create-channel)
+
+##### %channel
+[nest](#nest)
+
+###### %join
+```hoon
+=nest [%chat ~palnet-sampel %mychat]
+=action [%join group=[~sampel-palnet %mygroup]]
+
+:channels &channel-action [%channel nest action]
+```
+
+Join a channel. In this example, ~sampel-palnet is the group host and ~palnet-sampel is the channel host.
+###### %leave
+```hoon
+=nest [%chat ~palnet-sampel %mychat]
+=action [%leave ~]
+
+:channels &channel-action [%channel nest action]
+```
+
+Leave a channel
+###### %read
+```hoon
+=nest [%chat our %mychat]
+=action [%read ~]
+
+:channels &channel-action [%channel nest action]
+```
+
+Mark a channel as read.
+###### %read-at
+```hoon
+=nest [%chat our %mychat]
+=action [%read-at now]
+
+:channels &channel-action [%channel nest action]
+```
+
+Mark a channel as read up to a certain time.
+###### %watch
+```hoon
+=nest [%chat our %mychat]
+=action [%watch ~]
+
+:channels &channel-action [%channel nest action]
+```
+
+Watch a channel for unreads.
+###### %unwatch
+```hoon
+=nest [%chat our %mychat]
+=action [%unwatch ~]
+
+:channels &channel-action [%channel nest action]
+```
+
+Stop watching a channel for unreads.
+###### %post %add
+```hoon
+=nest [%chat our %mychat]
+=memo :*
+  ~[[%inline ~['hello world!']]]
+  author=our
+  sent=now
+==
+=kind-data [%chat ~]
+=essay [memo kind-data]
+=post-action [%add essay]
+=action [%post post-action]
+
+:channels &channel-action [%channel nest action]
+```
+
```

Comment:
A marker here for where commands start and maybe an explainer about what that means semantically.